lib/routes/route.js

```diff
@@ -613,6 +613,43 @@ module.exports = function(app, router) {
         });
     });
 
+    router.get("/user", function (req, res, next) {
+        handleSession(req.session, function (err, sessionObj) {
+            if (err) {
+                if (err.type === "userID") {
+                    req.session.userID = undefined;
+                    var err = {
+                        message: "Cannot perform action. User no longer exists."
+                    };
+                    return routeError(err, next, {
+                        errorID: "sessionUserNotFound",
+                        statusCode: 404,
+                        responseType: "json"
+                    });
+                } else {
+                    return routeError(err, next);
+                }
+            }
+            if (sessionObj.guest) {
+                var err = {
+                    message: "Cannot perform action. Not signed in."
+                };
+                return routeError(err, next, {
+                    errorID: "notSignedIn",
+                    statusCode: 400,
+                    responseType: "json"
+                });
+            }
+            else {
+                res.status(200);
+                return res.send({
+                    status: "success",
+                    message: "User Logged In"
```

Comment:
```suggestion
                    message: "User logged in"
```
and if you decide to keep the assertion for message, make sure it's updated there too.